flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/WebFrontendITCase.java

```diff
@@ -83,16 +78,16 @@ public static void initialize() throws Exception {
 		assertTrue("Unable to create temp directory", logDir.mkdir());
 		File logFile = new File(logDir, "jobmanager.log");
 		File outFile = new File(logDir, "jobmanager.out");
-		
+
 		Files.createFile(logFile.toPath());
 		Files.createFile(outFile.toPath());
-		
+
 		config.setString(JobManagerOptions.WEB_LOG_PATH, logFile.getAbsolutePath());
 		config.setString(ConfigConstants.TASK_MANAGER_LOG_PATH_KEY, logFile.getAbsolutePath());
 
 		cluster = new LocalFlinkMiniCluster(config, false);
 		cluster.start();
-		
+
```

Comment:
Sometimes, I also get this kinda stuff. But seems i didnt do anything relevant to this style.